src/commands/deploy-backend.ts

```diff
@@ -17,6 +17,7 @@ export default class DeployBackend extends BaseCommand {
     ...BaseCommand.commonFlags,
     region: flags.string({char: 'r', description: 'Region', default: 'us-west-2'}),
     subdomain: flags.string({char: 's', description: 'Subdomain'}),
+    deploymentMode: flags.string({char: 'm', description: 'Deployment Mode', default: 'graphql'})
```

Comment:
To be consistent with flags in other commands, this should be hyphenated as `deployment-mode` instead of `deploymentMode`.